cluster/log-dump/log-dump.sh

```diff
@@ -611,6 +621,11 @@ function dump_nodes_with_logexporter() {
     done
   fi
 
+  # If less than a certain ratio of the nodes got logexported, report an error.
+  if [[ $((${#NODE_NAMES[@]} - ${#failed_nodes[@]})) -lt $((${#NODE_NAMES[@]} * log_dump_expected_success_percentage / 100)) ]]; then
```

Comment:
dividing is always tricky - let's instead do:

100 * (node_names - failed_node) < node_names * percentage

Reply:
> dividing is always tricky

Especially tricky in bash which doesn't know anything about floating-point numbers ;) thanks for the remark, done.